### PR TITLE
Update Get-CsOnlineUser.md

### PR DESCRIPTION
**Pull request by isrumnon**

### CODE REVIEW

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 1]
```diff
@@ -94,25 +94,11 @@ Example 4 returns information only for user accounts that have been configured s
...
-```
-$MeetingPolicy = "Kiosk"
-$FilterString = 'TeamsMeetingPolicy -eq "{0}"' -f $MeetingPolicy
-Get-CsOnlineUser -Filter $FilterString
-```
-
-The commands shown in Example 6 filters all the online users with a certain TeamsMeetingPolicy assigned using a variable as filter input.
-
-To accomplish the task, the filter string is first constructed and resolved locally and then used by the Get-CsOnlineUser cmdlet.
+Examples above returns information for user accounts that have been assigned a designated phone number.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:101]:
```suggestion
Example 5 returns information for user accounts that have been assigned a designated phone number. 
```

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 2]
```diff
@@ -257,23 +244,27 @@ In Teams PowerShell Module version 3.0.0 and later, the **OnPremLineURI** attrib
 
 In the Teams PowerShell Module version 3.0.0 or later, the format of the AssignedPlan and ProvisionedPlan attributes has changed from XML to JSON array. Previous XML filters (For example, `-Filter "AssignedPlan -eq '<some-xml-string>'"`) will no longer work. Instead, you need to update your filters to use one of the following formats:
 
-- All users with an AssignedPlan that ends with "MCO": `-Filter "AssignedPlan -eq '*MCO'"`
 - All users with an AssignedPlan that starts with MCO: `-Filter "AssignedPlan -eq 'MCO*'"`
 - All users with an AssignedPlan that contains MCO: `-Filter "AssignedPlan -eq '*MCO*'"`
+- All users with an AssignedPlan that ends with "MCO": `-Filter "AssignedPlan -eq '*MCO'"`
+
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:250]:
```suggestion
```

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 3]
```diff
@@ -257,23 +244,27 @@ In Teams PowerShell Module version 3.0.0 and later, the **OnPremLineURI** attrib
...
 
 **Policy Attributes**:
 
 - PolicyProperty comparison works only when "Authority" is provided in the value. For ex: `-Filter "TeamsMessagingPolicy -eq '<Authority>:<Value>'"`
-"Authority" can contain any of these two values: Host or Tenant for a policy type (configurations that are provided by default are referred to as Host configurations while admin-created configurations are considered Tenant configurations).
+"Authority" can contain any of these two values: Host or Tenant for a policy type (configurations that are provided by default are referred to as Host configurations while admin-created configurations are considered Tenant configurations). Below are more examples:
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:255]:
```suggestion
"Authority" can contain any of these two values: Host or Tenant for a policy type (configurations that are provided by default are referred to as Host configurations while admin-created configurations are considered Tenant configurations). The following are more examples:
```

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 4]
```diff
@@ -245,6 +231,7 @@ In the Teams PowerShell Module version 3.0.0 or later, filtering functionality i
 - UserDirSyncEnabled
 - TeamsWorkLoadPolicy
 - VoiceRoutingPolicy
+- LastName (available after TPM 4.2.0)
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:234]:
```suggestion
- LastName (available in Teams PowerShell Module 4.2.0 and later)
```

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 5]
```diff
@@ -257,23 +244,27 @@ In Teams PowerShell Module version 3.0.0 and later, the **OnPremLineURI** attrib
...
+
 
 **Policy Attributes**:
 
 - PolicyProperty comparison works only when "Authority" is provided in the value. For ex: `-Filter "TeamsMessagingPolicy -eq '<Authority>:<Value>'"`
-"Authority" can contain any of these two values: Host or Tenant for a policy type (configurations that are provided by default are referred to as Host configurations while admin-created configurations are considered Tenant configurations).
+"Authority" can contain any of these two values: Host or Tenant for a policy type (configurations that are provided by default are referred to as Host configurations while admin-created configurations are considered Tenant configurations). Below are more examples:
+
+1. Filter "TeamsMessagingPolicy -eq 'Host:EduStudent'"
+2. Filter "TeamsMessagingPolicy -eq 'Tenant:TestDemoPolicy'"
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:258]:
```suggestion
- Filter "TeamsMessagingPolicy -eq 'Host:EduStudent'"
- Filter "TeamsMessagingPolicy -eq 'Tenant:TestDemoPolicy'"
```

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 6]
```diff
@@ -542,7 +533,7 @@ The following attributes are no longer relevant to Teams and have been dropped f
 - OnPremSamAccountName
 - CallerIdPolicy
 - Fax
-- LastName
+- LastName (available after TPM 4.2.0)
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:536]:
```suggestion
- LastName (available in Teams PowerShell Module 4.2.0 and later)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @isrumnon, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

File: skype/skype-ps/skype/Get-CsOnlineUser.md [hunk 7]
```diff
@@ -257,23 +244,26 @@ In Teams PowerShell Module version 3.0.0 and later, the **OnPremLineURI** attrib
...
 - In the Teams PowerShell Module version 3.0.0 or later, the output format of Policies has now changed from String to JSON type UserPolicyDefinition.
 
-*Dropped Filter operators*:
+*Change in Filter operators*:
 
 The following filter syntaxes have been modified in Teams PowerShell Module 3.0.0 and later:
 
 - -not, -lt, -gt: These operators have been dropped.
-- -ge, -ne:  These operators are not supported with policy properties.
+- -ge:  These operators are not supported with policy properties.
```
Comment (get-itips) [inline skype/skype-ps/skype/Get-CsOnlineUser.md:266]:
```suggestion
- -ge:  This operator is not supported with policy properties.
```

**Review by dariomws — APPROVED** [review]
I'm ok after @get-itips update.

**Review by tseward — APPROVED** [review]
Approved with @get-itips change requests.

**Comment by yogkumgit** [comment]
@isrumnon

**Comment by isrumnon** [comment]
@yogkumgit done! please merge

**Comment by yogkumgit** [comment]
@get-itips

**Comment by yogkumgit** [comment]
@tiburd Please merge this Pr. Thanks

**Pull request merged**